network/address_test.go

```diff
@@ -63,3 +80,22 @@ func TestAddress(t *testing.T) {
 		assert.Equal(t, str.Public, add.Public())
 	}
 }
+
+// just a temporary test case
+func TestDNSNames(t *testing.T) {
+	assert.True(t, validHostname("myhost.secondlabel.org"), "valid")
```

Comment:
It's not a big deal, but you don't need to specify a third argument for `assert` and `require`, looking at the output we already know if it's valid or not and where is the invalid line, so it does not bring any additional useful information.